test/edges/cubic-bezier-edge.test.ts

```diff
@@ -146,7 +146,9 @@ describe('CubicBezierEdge', function(): void {
           smooth: {
             roundness: 2,
           },
-          selfReferenceSize: 42,
+          selfReferenceSize: {
+            size: 42
+          },
```

Comment:
Why? Is this a breaking change? Number wouldn't work anymore?

Reply:
No it is not, that can be as it was, you are right.